src/cairo_run.rs

```diff
@@ -217,8 +217,8 @@ mod tests {
     #[test]
     fn write_binary_trace_file() {
         let program_path = Path::new("cairo_programs/struct.json");
-        let expected_trace_path = Path::new("cairo_programs/struct.trace");
-        let cairo_rs_trace_path = Path::new("cairo_programs/struct_rs.trace");
+        let expected_trace_path = Path::new("cairo_programs/trace_memory/cairo_trace_struct");
+        let cairo_rs_trace_path = Path::new("cairo_programs/trace_memory/struct_cleopatra.trace");
```

Comment:
Please change the name of the files as Cleopatra is no longer the official name

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs/pull/386/commits/76c7b1433769189d7d77dda9d368b1fd18eb384c